doc/vimspectorpy.txt

```diff
@@ -203,7 +203,23 @@ You could choose 'rxvt' or force 'xterm' using:
 
 >
   let g:vimspectorpy#launcher = "rxvt"
+
 <
+                          g:vimspectorpy#tmux#size
+
+Set the size of the split opened, when using the tmux launcher. Default is
+equivalent to
+
+  let g:vimspectorpy#tmux#size = 10
```

Comment:
To quote a block of ex-commands verbatim, place a greater than (>) character
at the end of the line before the block and a less than (<) character as the
first non-blank on a line following the block.

Reply:
Thansk for the quote, I now know where the vim-help is documented _and_ after some poking around figured out that I was fooled by some vim settings of mine hiding the syntax stuff unless I highlight the whole thing (took me 10 solid minutes to figure out why my formatting is different than what is already in the help file, even though seemingly they are the same ....). I also fixed the missing asteriks around the headers.